Day3/Kotlin/HammingDistance.kts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @author: d-l-mcbride
+ * @date: 10/12/2020
+ **/
+
+fun hammingDistance(first: String, second: String ):Int {
+    var distance = 0
+    for(i in first.indices) {
```

Comment:
Need to verify that the two strings are of the same length before checking for the hamming distance between the two strings.